plugins/resizimg/resize.with.canvas.js

```diff
@@ -11,10 +11,18 @@
         this.ctx = null;
         this.resizeimg = null;
 
+        /* jshint unused:vars */
         //function callback to do something when appen something
-        this.beforecanvasreplaced = function (canvas, image){};
-        this.presskeyesc = function (obj){};
-        this.presskeydelorcanc = function (obj){};
+        this.beforecanvasreplaced = function (canvas, image){
```

Comment:
I prefer camelCase names:

`beforeCanvasReplaced`

Also, there are some lint issues. And why this function is empty?

Reply:
This function is empty because It use to modify the image before it is restored in the HTML of the editor. For example, if you have some filter that allow you to create a version of your image already resized, you could modify the css class or the src of the image to do it. I leave it empty because is the default behavior: restore the image without something of special.